graphics/timg/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```

Comment:
please align the Portfile appropriately, and follow the modeline. In other words, spaces in multiples of four.

Also, please try to follow the usual order of fields - you can check other ports for guidance.

Reply:
I was kind of hoping the `port lint` command would have checked those items. That would make it easier for new contributors.